godot_project/project_workspace/custom_shapes/pyramid_mesh.gd

```diff
@@ -52,10 +52,6 @@ func get_height() -> float:
 	return height
 
 
-func request_update() -> void:
-	_update()
-
-
```

Comment:
Deleting this method means it doesn't work on 4.2 anymore because it's called in the setters

Reply:
Apart from that, this PR still works perfectly with 4.2 :+1:

Reply:
right! i should move this to the next PR